src/PerfView/HeapView/HeapDiagram.cs

```diff
@@ -696,10 +696,12 @@ void SetYScale(Renderer render)
 
             foreach (TraceGC gc in m_data.events)
             {
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
As mentioned above, one way or the other, I don't want most of these pragmas here.